controller/api/destination/watcher/prometheus.go

```diff
@@ -34,6 +34,68 @@ type (
 	}
 )
 
+var (
+	informer_lag_ms_buckets = []float64{
+		500,     // 500ms
+		1000,    // 1s
+		2500,    // 2.5s
+		5000,    // 5s
+		10000,   // 10s
+		25000,   // 25s
+		50000,   // 50s
+		100000,  // 1m 40s
+		250000,  // 4m 10s
+		1000000, // 16m 40s
+	}
+	endpointsInformerLag = promauto.NewHistogram(
+		prometheus.HistogramOpts{
+			Name:    "endpoints_informer_lag_ms",
+			Help:    "The amount of time between when an Endpoints resources is updated and when an informer observes it",
+			Buckets: informer_lag_ms_buckets,
+		},
+	)
```

Comment:
Can we somehow condition this declaration to whether endpointsslices is enabled or not? If it is, which is the default nowadays, I think these metrics will always show up empty.
... or we can leave this in and just wait till we're able to drop support for endpoints in favor of endpointslices altogether, which should happen pretty soon if not already.

Reply:
I think it's simplest to just leave them in.  It's just one histrogram without any additional labels, so I don't think it should be a problem.